### ISSUE TITLE
[feature request] Purchase services from merchant (no item transfer)

### ISSUE THREAD

**Issue by TheFinalPerry** [jopeek/fvtt-loot-sheet-npc-5e#336]
Would love a way to include services in merchant sheets. Namely things the players pay for that would not result in an item added to the players sheet.

Some examples:
- buying drinks and food at the bar
- paying for a room
- getting a spellcaster to cast something for you
- paying to copy a spell from a wizards spellbook

Ideally the merchant sheet could support physical items as well as services. Perhaps as a separate item category.

**Comment by jazz-at-CampTieDye** [comment]
Yes, this would be awesome. Maybe a way to add things as journal updates. So it keeps track of what you got where. (I know that would be insane, but heck still cool)

**Comment by DanielBoettner** [comment]
> Yes, this would be awesome. Maybe a way to add things as journal updates. So it keeps track of what you got where. (I know that would be insane, but heck still cool)

Already have this as a long term goal to add support for [Regions](https://gitlab.com/geekswordsman/regions)
to add info on items where they where bought/looted.

Regarding this specific feature request.

I like the idea (close to another suggestion of infinite items).

Will have to see how to add all the things needed for this.

- ignore them during loot
- allow buying (maybe looting) without removing from source
- when to transfer an item and how to handle duplicates (cause the copied spell should likely be copied to the player if bought)
- Easy and quick way for the GM to set/switch the state for an item.

**Comment by DanielBoettner** [comment]
#329 

> **Is your feature request related to a problem? Please describe.** Usually when making merchants I don't set item quantities and I leave everything at one.
> 
> **Describe the solution you'd like** I would like to be able to set a store with "infinite" quantities of items, or quantities not to decrease when a player buys an item. This would be useful to solve the problem described earlier as well as allow for other kinds of merchants, right now I'm using merchant sheets for staying in INNs, it would be useful to set bed quantities ![image](https://user-images.githubusercontent.com/43207603/160168319-55a74831-f57b-4ea0-92b2-558d0b29b7e0.png) (this meaning this INN has 4 available beds) but not reducing the quantity when bought.
> 
> **Describe alternatives you've considered** I considered when creating a merchant setting ridiculous quantities like 1000 of each item so players can buy them as they want, but this is not optimal as it requires additional amount of work and doesn't feel realistic.
